src/utils/potentialDupMsg/index.ts

```diff
@@ -0,0 +1,34 @@
+import { Message } from '@glif/filecoin-message'
+import BigNumber from 'bignumber.js'
+import { Message as GQLMessage } from '../../generated/graphql'
+import { isAddrEqual } from '../isAddrEqual'
+
+type ComparisonMessage = Pick<
+  GQLMessage,
+  'cid' | 'from' | 'to' | 'value' | 'params' | 'method'
+> & { height: string | number }
```

Comment:
This is added manually because `MessagePending` from graphql has a falsey `height`